<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="window">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 238, 246);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topBar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1,1,0">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
    background-color: rgb(233, 238, 246);
    border-radius: 17px;
    border: 2px solid rgb(255, 255, 255);
    padding: 5px 10px;
}

QPushButton:hover {
    border: 2px solid rgb(220, 220, 220);
    background-color: rgb(250, 250, 250);
}

QPushButton:pressed {
    border: 2px solid rgb(150, 150, 150);
    background-color: rgb(230, 230, 230);
}

QPushButton:focus {
    outline: none;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
    background-color: rgb(233, 238, 246);
    border-radius: 17px;
    border: 2px solid rgb(255, 255, 255);
    padding: 5px 10px;
}

QPushButton:hover {
    border: 2px solid rgb(220, 220, 220);
    background-color: rgb(250, 250, 250);
}

QPushButton:pressed {
    border: 2px solid rgb(150, 150, 150);
    background-color: rgb(230, 230, 230);
}

QPushButton:focus {
    outline: none;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newMediaButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
    background-color: rgb(233, 238, 246);
    border-radius: 17px;
    border: 2px solid rgb(255, 255, 255);
    padding: 5px 10px;
}

QPushButton:hover {
    border: 2px solid rgb(220, 220, 220);
    background-color: rgb(250, 250, 250);
}

QPushButton:pressed {
    border: 2px solid rgb(150, 150, 150);
    background-color: rgb(230, 230, 230);
}

QPushButton:focus {
    outline: none;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchMediaField">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 238, 246);
color: rgb(0, 0, 0);
border-radius: 11px;
padding-left: 6px;
padding-right: 6px;</string>
         </property>
         <property name="placeholderText">
          <string>Cerca</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>450</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="contentFrame">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="dialogContainer">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: qlineargradient(
        x1: 0, y1: 0,
        x2: 0, y2: 1,
        stop: 0 rgb(233, 238, 246),
        stop: 0.03 rgb(255, 255, 255),
        stop: 0.03 rgb(255, 255, 255),
        stop: 1 rgb(255, 255, 255)
    );</string>
         </property>
         <widget class="QWidget" name="createMediaPage"/>
         <widget class="QWidget" name="viewMediaPage"/>
         <widget class="QWidget" name="savePage"/>
         <widget class="QWidget" name="loadPage"/>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea{
	border:none;
}

QScrollBar:vertical {
    background: rgb(255, 255, 255);
    width: 10px;
    margin: 2px 0 2px 0;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: rgb(156, 176, 206);
    border-radius: 5px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: rgb(122, 137, 160);
}

QScrollBar::handle:vertical:pressed {
    background: rgb(93, 104, 122);
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    border: none;
    background: none;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none;
}

/* Scrollbar orizzontale */
QScrollBar:horizontal {
    background: rgb(255, 255, 255);
    height: 10px;
    margin: 0 2px 0 2px;
    border-radius: 5px;
}

QScrollBar::handle:horizontal {
    background: rgb(156, 176, 206);
    border-radius: 5px;
    min-width: 20px;
}

QScrollBar::handle:horizontal:hover {
    background: rgb(122, 137, 160);
}

QScrollBar::handle:horizontal:pressed {
    background: rgb(93, 104, 122);
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    border: none;
    background: none;
}

QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
    background: none;
}</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>400</width>
            <height>545</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="mediaGrid">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetFixedSize</enum>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
